apps/judicial-system/backend/src/app/modules/file/file.controller.ts

```diff
@@ -167,7 +167,7 @@ export class FileController {
     CaseNotCompletedGuard,
     CaseFileExistsGuard,
   )
-  @RolesRules(prosecutorRule, representativeRule)
+  @RolesRules(prosecutorRule, representativeRule, registrarRule, judgeRule)
```

Comment:
This is needed because judges and registrars can upload files in indictments and therefore they should be able to delete files as well

Reply:
You probably need to update the tests for this.